### PR TITLE
Fix archiver creates invalid archives

### PR DESCRIPTION
**Pull request by oliyg**
#95

### CODE REVIEW

**Review by oliyg — COMMENTED** [review]
await stream closed

**Comment by sibiraj-s** [comment]
Thanks @oliyg . Can you revert the lock file changes. I will merge this once that is done.

File: src/actions/archive.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ const archive = async (task, { logger }) => {
   const archive = archiver(format, options);
   archive.pipe(output);
 
+  const streamClose = async () => new Promise((resolve) => output.on('close', resolve));
```
Comment (sibiraj-s) [inline src/actions/archive.js:29]:
```suggestion
  const streamClose = () => new Promise((resolve) => output.on('close', resolve));
```

**Review by sibiraj-s — CHANGES_REQUESTED** [review]

**Comment by oliyg** [comment]
> Thanks @oliyg . Can you revert the lock file changes. I will merge this once that is done.

Now it's ok

**Comment by sibiraj-s** [comment]
Published in v6.1.6

**Pull request merged**